Assets/Scripts/Player.cs

```diff
@@ -49,9 +51,60 @@ public class Player : MonoBehaviour
 
 	private string openVideo;
 
-	void Start ()
+	private static class ConnectedDevices
```

Comment:
This class can definitely be public. Chance of using them in other scripts is very high. Might also be worth moving it to a different script file.